lib/home/widgets/recipes_screen.dart

```diff
@@ -93,7 +93,7 @@ class RecipesScreen extends StatelessWidget {
                                 'assignedFoodItemId': assignedFoodItemId,
                                 'foodItemDescription': foodItemDescription,
                                 'selectedScreen':
-                                    SelectedRecipeScreen.ingredientScreen
+                                    SelectedRecipeScreen.probeScreen
```

Comment:
This was with regards to standard recipes effectively only being used for their probe lists since ingredient information would be stored elsewhere - there is still more that would need to be done for this change but I think a discussion should be had before removing modified recipes from the app as requested by ICRISAT.

Reply:
Yup, as we discussed this internally today — we need to think this through and discuss with ICRISAT. For now the change to probeScreen is probably alright.